src/coreclr/hosts/unixcoreruncommon/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ project(unixcoreruncommon)
 
 add_compile_options(-fPIC)
 
+if(FEATURE_GDBJIT)    
```

Comment:
Now that you have moved the stuff to the hosting api, this is not needed here anymore